### PR TITLE
Hide Statistics MenuItem when necessary

### PR DESCRIPTION
**Pull request by mic4ael**

### CODE REVIEW

File: piwik/indico_piwik/controllers.py [hunk 1]
```diff
@@ -15,15 +15,24 @@
 # along with Indico; if not, see <http://www.gnu.org/licenses/>.
 
 from flask import jsonify, request
+from werkzeug.exceptions import Forbidden
 
+from flask_pluginengine import current_plugin
```
Comment (ThiefMaster) [inline piwik/indico_piwik/controllers.py]: that's thirdparty

File: piwik/indico_piwik/controllers.py [hunk 2]
```diff
@@ -15,15 +15,24 @@
...
                                   ReportVisitsPerDay)
 from indico_piwik.views import WPStatistics
 
 
-class RHStatistics(RHManageEventBase):
+class RHPiwikBase(RHManageEventBase):
+    def _process_args(self):
+        RHManageEventBase._process_args(self)
+        if not current_plugin.settings.get('site_id_events'):
+            raise Forbidden
```
Comment (ThiefMaster) [inline piwik/indico_piwik/controllers.py]: NotFound

File: piwik/indico_piwik/controllers.py [hunk 3]
```diff
@@ -32,25 +41,25 @@ def _process(self):
...
 
     def _process_args(self):
-        RHManageEventBase._process_args(self)
+        RHPiwikBase._process_args(self)
         self._report_params = {'start_date': request.args.get('start_date'),
                                'end_date': request.args.get('end_date')}
 
 
-class RHApiEventBase(RHApiBase):
+class RHApiEventBase(RHPiwikBase):
```
Comment (ThiefMaster) [inline piwik/indico_piwik/controllers.py]: look at the inheritance chain again.. you don't need to change this

**Comment by mic4ael** [comment]
Updated

**Pull request merged**